docs/model_fields.rst

```diff
@@ -695,6 +695,45 @@ Or with attribute assignment to a model::
         any conflicts.
 
 
+.. _enum-fields:
+
+-----------
+Enum Fields
+-----------
+
+You can take advantage of MySQL's ``ENUM`` type through the ``EnumField``
+provided by django-mysql. Using ``ENUM`` provides the benefit of storing
+recurring strings in a very space efficient manner. MySQL will store the
+enumeration value in each row as an integer value and return results as
+the corresponding strings in queries.
+
+Specifying the range of values available in an EnumField must be specified
+in the `choices` argument as a tuple or list::
```

Comment:
escaping is with two backticks in RST. Have you managed to compile the docs locally? `cd docs` , `make html` or `make livehtml`